metricbeat/module/coredns/stats/_meta/fields.yml

```diff
@@ -0,0 +1,65 @@
+- name: metrics
+  type: group
+  description: >
+    Contains statistics related to the coreDNS service
+  release: beta
+  fields:
+    - name: panic.count.total
+      type: long
+      description: >
+         Total number of panics
+    - name: dns.request.count.total
+      type: long
+      description: >
+         Total query count
+    - name: dns.request.duration.seconds
```

Comment:
`.sec` is one of our exceptions: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html